manifests/repo/dnfmodule.pp

```diff
@@ -0,0 +1,24 @@
+# @summary Manage the DNF module
+#
+# On EL8+ DNF can manage modules.
+# This is a method of providing multiple versions on the same OS.
+# Only one DNF module stream can be active at the same time.
+#
+# @param ensure
+#   Value of ensure parameter for nodejs dnf module package
+#
+# @param module
+#   Name of the nodejs dnf package
+#
+# @api private
+class nodejs::repo::dnfmodule (
+  String[1] $ensure = $nodejs::repo_version,
+  String[1] $module = 'nodejs',
+) {
+  package { 'nodejs dnf module':
+    ensure      => $ensure,
+    name        => $module,
+    enable_only => true,
```

Comment:
I'm not sure if this is still needed. I've dropped it in other places so solve upgrading issues.

Reply:
not where I've copied it from XD

https://github.com/voxpupuli/puppet-redis/blob/ee9956b10586284f3cbecc26baa2b79f35da0fc5/manifests/dnfmodule.pp#L21